scripts/scylla-get.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+#
+# A platform agnostic Scylla server installation script.
+#
+# To install Scylla, run the following command:
+#
+# curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/scylladb/scylla/master/scripts/scylla-get.sh | sudo bash
+#
+# This file is open source software, licensed to you under the terms
+# of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+# distributed with this work for additional information regarding copyright
+# ownership.  You may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+set -euo pipefail
+
+main() {
+  require_cmd curl
+
+  SCYLLA_VERSION_RAW=$(curl -s https://repositories.scylladb.com/scylla/check_version?system=scylla)
+
+  SCYLLA_VERSION=$(echo $SCYLLA_VERSION_RAW | sed -e "s/.*version\":\"\(.*\)\".*/\1/g")
+
+  SCYLLA_RELEASE=$(echo $SCYLLA_VERSION | sed -e "s/\([[:digit:]]\+.[[:digit:]]\+\).*/\1/g")
+
+  while [ $# -gt 0 ]; do
+    case "$1" in
+      "-h" | "--help")
+        usage
+	exit 1
+        ;;
+      "--scylla-version")
+        SCYLLA_VERSION="$2"
+        shift 2
+        ;;
+      *)
+        usage
+	exit 1
+        ;;
+    esac
+  done
+
+  ARCH=$(uname -m)
+
+  if [[ "$ARCH" != "x86_64" ]]; then
+    echo "Architecture $ARCH is not supported by this installer."
```

Comment:
Eventually it could just fall back to the tarball installer and work for most systems, right?

Reply:
Yes, I think falling back to the tarball makes sense as a future improvement.